src/main/java/io/jenkins/plugins/appcenter/task/internal/DistributeResourceTask.java

```diff
@@ -99,8 +101,15 @@ private String parseReleaseNotesFromFile(@Nonnull String pathToReleaseNotes) {
 
     }
 
+    private static BuildInfo CreateBuildInfo(@Nonnull UploadRequest request) {
+        if (request.branchName == null && request.commitHash == null) return null;
+        return new BuildInfo(request.branchName,request.commitHash,null);
```

Comment:
Just a quick question do you know what will happen if the user supplies _both_ the branch name and the commit hash? Is that possible? Allowed? Should we protect against this earlier in the transaction?

Reply:
Ahh ignore me this seems only be purely informational like the release notes. So I guess this won't fail but will be confusing for the user. But then that's their fault. Hahaha!